packages/stash-cli/src/commands/describe-collection.ts

```diff
@@ -25,22 +25,47 @@ const command: GluegunCommand = {
       process.exit(1)
     })
 
+    function exitWithUsage(level: "info" | "error" = "info"): never {
+      print[level](
+        "Usage: stash describe-collection <collection-name> [--profile <profile>] [--include-meta] [--json] [--help]"
+      )
+      print.newline()
+      print[level]("Display details on identifiers and indexes for the given collection.")
+      print.newline()
+      print[level]("Example: stash describe-collection movies")
+      process.exit(level === "info" ? 0 : 1)
+    }
+
+    if (parameters.options.help) {
+      exitWithUsage()
+    }
+
     try {
       const stash = await Stash.connect(profile)
       const collectionName = parameters.first
       if (collectionName === undefined) {
         print.error("No collection name specified.")
-        process.exit(1)
+        print.newline()
+        exitWithUsage("error")
       }
 
       const collection = await stash.loadCollection(collectionName)
       const mappings = {}
 
       Object.entries(collection.schema.mappings).forEach(([indexName, mapping]) => {
+        const meta = parameters.options.includeMeta
+          ? {
+              $prpKey: collection.schema.meta[indexName]!.$prpKey.toString("hex"),
+              $prfKey: collection.schema.meta[indexName]!.$prfKey.toString("hex"),
+              $indexId: collection.schema.meta[indexName]!.$indexId,
+            }
+          : {}
+
         mappings[indexName] = {
           indexType: mapping.kind,
           fields: describeFields(mapping),
           operators: describeOperators(mapping),
+          ...meta,
```

Comment:
I wonder if it's worth renaming `mappings` here to be `indexes` like in the docs. It's not a big deal from the Rust side since I can add an alias for "mappings" but it wouldn't be possible to take the output of a describe-collection and create another collection from it for example.

Reply:
Ah yeah, good point. It's a breaking change but it makes sense to be consistent and I don't think anything else is consuming the output from this command anyway.

Reply:
Easier to change it sooner rather than later as well.